src/components/LeftPanel.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+const LeftPanel: React.FC = () => {
```

Comment:
So LeftPanel itself is a component of type React.FC (functional component), but it actually only calls three links.

Reply:
Yes indeed